test/libraries/SafeCast.t.sol

```diff
@@ -3,10 +3,10 @@ pragma solidity ^0.8.20;
 
 import {Test} from "forge-std/Test.sol";
 import {Vm} from "forge-std/Vm.sol";
-import {SafeCast} from "src/libraries/SafeCast.sol";
+import {SafeCast} from "../../src/libraries/SafeCast.sol";
 
 contract SafeCastTest is Test {
-    function testToUint160(uint256 x) public {
+    function test_fuzz_toUint160(uint256 x) public {
```

Comment:
the others say like `_fromUint256`

Reply:
oh thats just cuz theres 2 `toInt128`s, so i differentiated them by adding the `from`s. 
For `toUint160` and `toUint256` I didn't add the `from`s, but I can if it makes it easier